hackathon_site/event/jinja2/event/landing.html

```diff
@@ -27,292 +29,296 @@
+                    <h4 class="center white-text">Saturday</h4>
+
+                    <div class="scheduleRow">
+                        <p class="lightText scheduleTime">9AM</p>
+                        <p class="white-text">Participant Registration</p>
+                    </div>
+                    <div class="scheduleRow">
+                        <p class="lightText scheduleTime">10AM</p>
+                        <p class="white-text">Opening Ceremonies</p>
+                    </div>
+                    <div class="scheduleRow">
+                        <p class="lightText scheduleTime">11AM</p>
+                        <p class="white-text">
+                            <strong>Hacking Begins</strong>
+                        </p>
+                    </div>
+                    <div class="scheduleRow">
+                        <p class="lightText scheduleTime">Various</p>
+                        <p class="white-text">Workshops & Activities</p>
+                    </div>
                 </div>
-            </div>
 
-            <div class="col s12 m5 l6">
-                <h4 class="center">Sunday</h4>
+                <div class="col s12 m5 l6">
+                    <h4 class="center white-text">Sunday</h4>
 
-                <div class="scheduleRow">
-                    <p class="scheduleTime">11AM</p>
-                    <p><strong>Hacking Ends!</strong></p>
-                </div>
-                <div class="scheduleRow">
-                    <p class="scheduleTime">12PM</p>
-                    <p>Judging</p>
-                </div>
-                <div class="scheduleRow">
-                    <p class="scheduleTime">3PM</p>
-                    <p>Closing Ceremonies</p>
-                </div>
-                <div class="scheduleRow">
-                    <p class="scheduleTime">4PM</p>
-                    <p>End</p>
+                    <div class="scheduleRow">
+                        <p class="lightText scheduleTime">11AM</p>
+                        <p class="white-text"><strong>Hacking Ends!</strong></p>
+                    </div>
+                    <div class="scheduleRow">
+                        <p class="lightText scheduleTime">12PM</p>
+                        <p class="white-text">Judging</p>
+                    </div>
+                    <div class="scheduleRow">
+                        <p class="lightText scheduleTime">3PM</p>
+                        <p class="white-text">Closing Ceremonies</p>
+                    </div>
+                    <div class="scheduleRow">
+                        <p class="lightText scheduleTime">4PM</p>
+                        <p class="white-text">End</p>
+                    </div>
                 </div>
             </div>
-        </div>
-        <div class="m12 center">
-            <a href="#" class="btn-large waves-effect waves-light colorBtn" rel="noopener" target="_blank">Full schedule</a>
+            <div class="m12 center">
```

Comment:
Given that this is going live soon, may want to hide / remove this button until it actually does something

Reply:
I asked logistics for a docs to link and they'll send it soon

Reply:
eh i'll put it back once they get back to me